spec/factories.rb

```diff
@@ -52,7 +52,9 @@
     end
 
     after(:build) do |organization, evaluator|
-      create_list(:user, evaluator.users_count, organizations: [organization])
+      if evaluator.users.count < evaluator.users_count
+        create_list(:user, evaluator.users_count, organizations: [organization])
```

Comment:
I added this check because this is adding random extra users to the organization. 

This will break the functionality of [`Organization#github_client`](https://github.com/education/classroom/blob/master/app/models/organization.rb#L37:L40) since the `access_token` of the randomly created user is not valid, which will make the API calls return 401.